include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -7280,7 +7294,14 @@
           "enumerant" : "ShadingRateNV",
           "value" : 5291,
           "capabilities" : [ "Shader" ],
-          "extensions" : [ "SPV_NV_shading_rate" ],
+          "extensions" : [ "SPV_NV_shading_rate", "SPV_EXT_fragment_invocation_density" ],
+          "version" : "None"
+        },
+        {
```

Comment:
I'd put the FragmentDensityEXT spelling first, so swap this element with the previous.  That would match the other enums.